core/modules/installation/templates/installstep1.html.php

```diff
@@ -63,6 +63,16 @@
             More information is available at <a href="http://php.net/manual/en/book.mbstring.php" target="_blank">php.net</a>
             </div>
         <?php endif; ?>
+        <?php if ($dom_ok): ?>
+            <div class="install_progress prereq_ok"><?php echo image_tag('images/action_ok.png', array(), true); ?>PHP DOMDocument class is available</div>
+        <?php else: ?>
+            <div class="install_progress prereq_fail">
+                <b>PHP Class "DOMDocument" is not available</b><br>
+                The Bug Genie 3 requires the PHP Class "DOMDocument". For most distributions, the extension which contains this class is called "php-xml".<br/>
```

Comment:
Technically, we're at version 4.
```suggestion
                The Bug Genie 4 requires the PHP Class "DOMDocument". For most distributions, the extension which contains this class is called "php-xml".<br/>
```

Reply:
I just did a copy'n'paste from the previous row, I guess it would make more sense just to remove the version number from the whole file, no?

Reply:
Yes, that makes total sense :)